README.md

```diff
@@ -1,113 +1,156 @@
-# io-wallet
-EUDI Wallet and Italian Wallet implementation for App IO
+# IO Wallet - Monorepo
 
-## Related repositories:
-- 🪪 [io-react-native-wallet](https://github.com/pagopa/io-react-native-wallet)
-- 🔐 [io-react-native-jwt](https://github.com/pagopa/io-react-native-jwt)
-  
-## Prerequisites
+### Introduction
 
-In order to run the applications locally you need the following tool installed on your machine.
+Welcome! 😊
 
-- `Node.js`
-- `yarn`
+This is the `io-wallet` repository, a monorepo that use [NodeJS](https://nodejs.org/), [TypeScript](https://www.typescriptlang.org/), [fp-ts](https://gcanti.github.io/fp-ts/) and different [Azure SDKs](https://azure.github.io/azure-sdk/#javascript) for the IO Wallet app.
 
-The preferred way to set up the local environment is using [nodenv](https://github.com/nodenv/nodenv) to manage `Node.js` installation and `corepack` (included with `Node.js`) to manage the installation of `yarn`.
-Please refer to `.node-version` for the actual runtime version used.
+A monorepo is a single repository containing multiple different projects, with well-defined relationships and responsabilities. The `io-wallet` monorepo contains two main projects under the `app/` folder, present in the root directory of the monorepo:
 
+- `io-wallet-support-func`: it contains all functionalities for assistance and support. It's a simple backend.
+- `io-wallet-user-func`: it contains all functionalities used by the IO Wallet app end users. It's a complex backend.
 
```

Comment:
I would remove the last two sentences, as 'simple' and 'complex' are subjective adjectives